src/network/ssl/socket.c

```diff
@@ -279,8 +284,14 @@ match_uri_host_ip(const char *uri_host,
 	 * network byte order.  */
 	switch (ASN1_STRING_length(cert_host_asn1)) {
 	case 4:
+#ifndef win32
 		return inet_aton(uri_host, &uri_host_in) != 0
 		    && memcmp(cert_host_addr, &uri_host_in.s_addr, 4) == 0;
+#else
```

Comment:
The same as above applies here
#ifdef HAVE_INET_PTON
...
#else
...
#endif

Reply:
You're right. This is better.